Moodlet/app/src/main/java/com/cmput3owo1/moodlet/services/MoodEventService.java

```diff
@@ -0,0 +1,92 @@
+package com.cmput3owo1.moodlet.services;
```

Comment:
This class should include the use of the `DatabaseUtil` class. I think it would be better to start the implementation of the `DatabaseUtil` class now and include its use in the `MoodEvenService` class. This would require less refactoring of the code later on. On that note, you should just implement the needed functionality in the `DatabaseUtil` class, and we'll add to it as more of the functionality becomes needed.